src/main/java/org/jabref/gui/groups/GroupTree.css

```diff
@@ -102,8 +102,18 @@
     -fx-translate-x: -0.4em;
 }
 
-#newGroupButton {
-    -fx-padding: 0.1em 1.5em 0.1em 1.5em;
+#addNewGroup:active {
+    -fx-border-width: 0px;
+    -fx-background-color: -jr-theme;
+    -fx-padding: 0.5em;
+    -fx-text-fill: -jr-white;
+}
+
+#addNewGroup:inactive {
+    -fx-border-width: 0px;
+    -fx-background-color: -jr-icon-background-active;
+    -fx-padding: 0.5em;
+    -fx-text-fill:-jr-black;
```

Comment:
Hi, does this work with the black theme?

Reply:
@calixtus, sorry may you help to clarify your question? I am not sure what you mean by dark theme. I only used the colors available in `Base.css`.

Reply:
It would be great if you test your changes with the dark theme (preferences -> appearance -> dark theme)

Reply:
@calixtus, per checking dark them works properly on the newly implemented change. Kindly see attached file.
<img width="414" alt="Screen Shot 2021-06-08 at 7 28 36 pm" src="https://user-images.githubusercontent.com/7042670/121169278-fcfab000-c892-11eb-8b80-3d083d7ba161.png">